csrankings-n.csv

```diff
@@ -291,6 +291,7 @@ Ni Trieu,Arizona State University,https://nitrieu.github.io,esITaOoAAAAJ
 Nian-Feng Tzeng,University of Louisiana - Lafayette,https://people.cmix.louisiana.edu/tzeng,6zq-c5cAAAAJ
 Niangjun Chen,SUTD,https://istd.sutd.edu.sg/people/faculty/niangjun-chen,mOmPukcAAAAJ
 Nianwen Xue,Brandeis University,http://www.cs.brandeis.edu/~xuen,KlVbOkMAAAAJ
+Nianyi Li,Clemson University,https://www.clemson.edu/cecas/departments/computing/people/faculty/computer_science.html,3SqliH4AAAAJ

```

Comment:
Needs to be the faculty member's home page.

Reply:
Updated.